### PR TITLE
Add blocks hydration scripts to micro-frontend

### PR DESCRIPTION
**Pull request by karimkawambwa**
## Description

This will standardize frontend with WYSIWYG.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: wp-content/plugins/hurumap/api.php [hunk 1]
```diff
@@ -119,15 +125,22 @@ function get_charts($request)
...
     $placeholders = implode(', ',  array_fill(0, $published == null ? 2 : 1, '%s'));
     $values = $published === null ? [0, 1] : [$published == '1'];
 
-    $hurumap = $wpdb->get_results($wpdb->prepare("SELECT * FROM {$wpdb->base_prefix}hurumap_charts where published IN ({$placeholders}) order by created_at desc", $values));
-    $flourish = $wpdb->get_results($wpdb->prepare("SELECT * FROM {$wpdb->base_prefix}flourish_charts where published IN ({$placeholders}) order by created_at desc", $values));
-    $sections = $wpdb->get_results($wpdb->prepare("SELECT * FROM {$wpdb->base_prefix}chart_sections where published IN ({$placeholders}) order by `order` asc", $values));
-    $response = new WP_REST_Response(array('hurumap' => $hurumap, 'flourish' => $flourish, 'sections' => $sections));
+    if ($chart_id) {
+        $hurumap = $wpdb->get_results($wpdb->prepare("SELECT * FROM {$wpdb->base_prefix}hurumap_charts where published = 1 and id = %s", [$chart_id]));
+        $section = $wpdb->get_results($wpdb->prepare("SELECT * FROM {$wpdb->base_prefix}chart_sections where published = 1 and id = %s", [$hurumap->section]));
```
Comment (kilemensi) [inline wp-content/plugins/hurumap/api.php]: Shouldn't we check if there is valid `$hurumap` before trying to fetch a `$section` using `$hurumap->section`?

File: wp-content/plugins/hurumap/src/FlourishBlock/Save.js [hunk 2]
```diff
@@ -5,14 +5,15 @@ function Save({ attributes }) {
...
+      style={{ width: attributes.chartWidth || '100%' }}
       data-chart-id={attributes.chartId}
       data-chart-title={attributes.title}
       data-chart-description={attributes.description}
       data-show-insight={attributes.showInsight}
       data-insight-title={attributes.insightTitle}
       data-insight-summary={attributes.insightSummary}
       data-data-link-title={attributes.dataLinkTitle}
-      data-analysis-link-href={`/profiles/${attributes.analysisCountry}`}
+      data-analysis-country={attributes.analysisCountry}
```
Comment (kilemensi) [inline wp-content/plugins/hurumap/src/FlourishBlock/Save.js:16]: Can we be a bit more explicit here: `country-slug` or `country-code`

**Review by kilemensi — APPROVED** [review]
:rocket: @karimkawambwa

File: wp-content/plugins/hurumap/src/FlourishBlock/Save.js (hunk 2 above)
Comment (karimkawambwa) [reply wp-content/plugins/hurumap/src/FlourishBlock/Save.js:16]: @KhadijaMahanga

**Pull request merged**